projects/addon-doc/components/page/page.component.ts

```diff
@@ -44,6 +47,7 @@ import {TuiDocPageTabConnector} from './page-tab.directive';
     providers: PAGE_PROVIDERS,
 })
 export class TuiDocPage {
+    protected readonly tabs = inject(TUI_DOC_TABS)(inject(ActivatedRoute).snapshot);
```

Comment:
Update this line 
https://github.com/taiga-family/taiga-ui/blob/8217c9030781b30ae6102f57dc4d13f64199f6c1/projects/addon-doc/components/page/tests/page.component.spec.ts#L14

to fix unit tests error:

```
addon-doc:  FAIL  projects/addon-doc/components/page/tests/page.component.spec.ts
addon-doc:   ● TuiDocPageComponent › should create
addon-doc:     NullInjectorError: R3InjectorError(Standalone[TuiDocPage])[ActivatedRoute -> ActivatedRoute -> ActivatedRoute]: 
addon-doc:       NullInjectorError: No provider for ActivatedRoute!
```